### DISCUSSION TITLE
v2.6.5

### DISCUSSION THREAD

**Discussion by mashb1t** [mashb1t/Fooocus#74] [Announcements]
## What's Changed
* fix: resolve colab unsupported image type issue by @mashb1t in https://github.com/lllyasviel/Fooocus/pull/3506

**Full Changelog**: https://github.com/mashb1t/Fooocus/compare/v2.6.4...v2.6.5

<hr /><em>This discussion was created from the release <a href='https://github.com/mashb1t/Fooocus/releases/tag/v2.6.5'>v2.6.5</a>.</em>

**Comment by HDL34** [comment]
Hi, I updated the English json language file. It was missing a lot of information regarding the interface content. I also made a French version. But I still have some things I haven't been able to translate because I don't know how they are coded.
[fr.json](https://github.com/user-attachments/files/21060043/fr.json)
[en.json](https://github.com/user-attachments/files/21060044/en.json)
